CMakeLists.txt

```diff
@@ -77,10 +78,14 @@ if(UAGENT_P2P_PROFILE)
 endif()
 
 if(UAGENT_FAST_PROFILE)
-    set(_fastdds_version 2.0.1)
-    set(_fastdds_tag 2.0.x)
+    if(UAGENT_FAST_PROFILE AND NOT AGENT_USE_SYSTEM_FASTDDS)
+        set(_fastdds_version 2.0.0)
```

Comment:
Are we sure that setting `_fastdds_version` to  `2.0.0` will look for the minimum version [here](https://github.com/eProsima/Micro-XRCE-DDS-Agent/blob/38de6ecc739b9104bea5a582bbabb51cd4adbbb7/CMakeLists.txt#L135)?

I see [here](https://cmake.org/cmake/help/v3.0/command/find_package.html) that the format is: `major[.minor[.patch[.tweak]]]` so maybe we should use `set(_fastdds_version 2)`. Can you test it locally before fixing it in the PR?